src/js/utils/Select.jsx

```diff
@@ -107,7 +107,7 @@ class Select extends Component {
       multi, placeholder, showLabelTooltip, value, defaultPlaceholder,
     } = this.props;
 
-    if (value?.translatedName) {
+    if (value?.translatedName || value?.hasTranslatedName) {
```

Comment:
Could you show an exact example that this change is needed? I think we should rather stick to only `value?.hasTranslatedName`. 
Since `hasTranslatedName` is defined as this:
```
hasTranslatedName: !!obj.translatedName,
```
there should be no way that the `hasTranslatedName` is `true` and the `value.translatedName` is false, because if the `value.translatedName` would be `false`, the `hasTranslatedName` should also be `false`.

Reply:
Agreed. I'm also thinking we might benefit from having these convenience properties (hasDisplayName, hasBrandNames, hasAlternateNames, etc) returned by the API but I think that needs a bit more thinking.